src/Microsoft.AspNetCore.Localization/RequestLocalizationOptions.cs

```diff
@@ -109,5 +109,53 @@ public RequestCulture DefaultRequestCulture
         /// </list>
         /// </summary>
         public IList<IRequestCultureProvider> RequestCultureProviders { get; set; }
+
+        
+        /// <summary>
+        /// Adds the set of the supported cultures by the application.
```

Comment:
I think it's a good idea to add the links using `<see cref="CultureInfo.CurrentCulture"/>` and `<see cref="CultureInfo.CurrentUICulture"/>` to these. These two concepts are frequently misunderstood.

Reply:
Can you elaborate where should I reference them? Because the builder methods are using `SupportedCultures` & `SupportedUICultures`